azure-pipelines.yml

```diff
@@ -66,7 +66,7 @@ stages:
     parameters:
         platform:
           name: 'Managed'
-          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
+          container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0'
```

Comment:
It doesn't feel right to be running our source-build legs on mariner because it is not a distro that source-build is running on.  I think stream9 would be a better choice here.